doc/userdoc/guides/index.rst

```diff
@@ -14,6 +14,7 @@ Here you can find details on a variety of topics in NEST.
     :maxdepth: 1
     :caption: In depth guides to NEST
 
+    built-in_timers
```

Comment:
Would it make sense to move this rather technical point further down in the list of guides?